.bazelversion

```diff
@@ -1,2 +1 @@
-6.4.0
-# NOTE: Bazel 7.0 errors out on 'buildifier_lint_check' with a hint to 'go_register_toolchains' in 'setup.bzl'
+7.0.0
```

Comment:
Why not go for 7.1.1?

Reply:
>Why not go for 7.1.1?

done [c6626a5](https://github.com/eclipse-iceoryx/iceoryx/pull/2242/commits/c6626a5559184f70b33607ea33140f5d42291ab5)